frontend/src/components/ScheduledEventsTableRow/ScheduledEventsTableRow.tsx

```diff
@@ -10,18 +11,20 @@ export interface ScheduledEventsTableRowProps {
   repeat: string;
   interval_type: string;
   start_date: string;
+  onClick?: () => void;
```

Comment:
This one should take in the event, or id & name ... i.e. `onClick?: (event: ScheduledEvent) => void;`